iphone/TitaniumKit/TitaniumKit/Sources/API/TiViewProxy.m

```diff
@@ -1318,8 +1318,7 @@ - (BOOL)viewInitialized
 
 - (BOOL)viewReady
 {
-  return view != nil && !CGRectIsEmpty(view.bounds) && !CGRectIsNull(view.bounds) &&
-      [view superview] != nil;
+  return (view != nil) && ([view superview] != nil);
```

Comment:
This is just me being super nitpicky, but the brackets are not really required here.